samples/timelapse-sample/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+set(TARGET timelapse-sample)
+SET(CORE_FILES
+	timelapse_summary.cpp
+)
+SET(CORE_FILES_H
+	timelapse_summary.h
+)
+SET(SOURCE
+  ${CORE_FILES}
+  ${CORE_FILES_H}
+)
+set(UT_FILE
+	timelapse_summary_test.cpp
+)
+find_library(APRA_PIPES_LIB NAMES aprapipes)
+add_library(example_aprapipes STATIC ${SOURCE})
+add_executable(runner_test ${UT_FILE})
+target_include_directories ( example_aprapipes PRIVATE
```

Comment:
call it samples

Reply:
Done.

Reply:
Renaming it to timelapsesample would be better as there will be multiple samples with this name.